alacritty_terminal/src/input.rs

```diff
@@ -575,6 +578,21 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
         }
     }
 
+    fn is_expand_selection(
```

Comment:
Tiny nitpick, feel free to ignore.

Would this make more sense as `should_expand_selection`?

Reply:
I do like `should_expand_selection` better.